### PR TITLE
[JU-16 ] message -package -user and  account

### PR DESCRIPTION
**Pull request by emre9ybar**

### CODE REVIEW

File: src/main/java/org/example/account/CheckingAccount.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package org.example.account;
+
+import org.example.user.User;
+import org.example.user.UserService;
+
+public class CheckingAccount{
```
Comment (furkanErenoglu) [inline src/main/java/org/example/account/CheckingAccount.java:6]: Burdan kaldırmalısın

**Review by furkanErenoglu — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/messages/AccountMessage.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package org.example.messages;
+
+public class AccountMessage {
+
```
Comment (furkanErenoglu) [inline src/main/java/org/example/messages/AccountMessage.java:4]: kayıtta bahsettiğim gibi güncellmelisin ayrıca message package ı oluşturmaya gerek yok

**Review by furkanErenoglu — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/messages/AccountMessage.java (hunk 2 above)
Comment (emre9ybar) [reply src/main/java/org/example/messages/AccountMessage.java:4]: nasıl oluşturalım ?

**Comment by emre9ybar** [comment]
public static String FIND_ACCOUNT="Account found";
    public static String NOT_FOUND_ACCOUNT = "Account not found.";
    public static String CREATE_SUCCESS = "Account succesfully created.";
    public static String UPDATE_SUCCESS = "Account succesfully updated.";
    public static String READ_SUCCESS = "Account details retrieved successfuly";

 public static String FIND_USER = "User found.";
    public static String NOT_FOUND_USER = "User not found.";
    public static String CREATE_SUCCESS = "User successfully created.";
    public static String UPDATE_SUCCESS = "User successfully updated.";
    public static String READ_SUCCESS = "User details retrieved successfully";

**Pull request closed without merging**